src/main/java/org/isf/medicalstockward/model/MedicalWard.java

```diff
@@ -45,45 +46,51 @@
 @AttributeOverride(name = "lastModifiedDate", column = @Column(name = "MDSRWRD_LAST_MODIFIED_DATE"))
 public class MedicalWard extends Auditable<String> implements Comparable<Object> {
 
-	@EmbeddedId 
+	@EmbeddedId
 	MedicalWardId id;
-	
-	@Column(name="MDSRWRD_IN_QTI")
+
+	@Column(name = "MDSRWRD_IN_QTI")
 	private float in_quantity;
-	
-	@Column(name="MDSRWRD_OUT_QTI")
+
+	@Column(name = "MDSRWRD_OUT_QTI")
 	private float out_quantity;
-	
+
+	/**
+	 * Lock control
+	 */
+	@Version
+	@Column(name = "MDSRWRD_LOCK")
+	private Integer lock;
```

Comment:
why is this an `Integer`?; isn't `int` OK and better as it is a primitive and not an object.

Reply:
You're right. It was even making tests failing.